resources/views/layouts/app.blade.php

```diff
@@ -16,7 +16,8 @@
     </head>
     <body class="antialiased relative grid grid-cols-1 min-h-screen bg-gray-100 dark:bg-gray-900 selection:bg-green-300 selection:text-black">
         <div class="mx-auto p-6 lg:p-8">
-            <img src="/cronikl.svg" class="w-64 mb-8 pointer-events-none select-none" />
+            <img src="/cronikl.svg" class="w-64 mb-8 pointer-events-none select-none block dark:hidden" />
+            <img src="/cronikl-white.svg" class="w-64 mb-8 pointer-events-none select-none hidden dark:block" />
```

Comment:
This will do for now, but...

If the one SVG could be coloured with the `fill:` classes, that would even nicer than having to swap 2 separate image files' visibility

Reply:
Yepyepyep agreed, will sort that at some point.